ob1k-core/src/main/java/com/outbrain/ob1k/client/targets/SimpleTargetProvider.java

```diff
@@ -11,11 +13,12 @@
  * @author eran 6/21/15.
  */
 public class SimpleTargetProvider implements TargetProvider {
-
+  private static final Logger logger = LoggerFactory.getLogger(SimpleTargetProvider.class);
   private final String target;
 
   public SimpleTargetProvider(final String target) {
     this.target = Preconditions.checkNotNull(target, "target must not be null");
+    logger.info("Target url: {}", target);
```

Comment:
I honestly don't see any value it in - maybe, maybe debug, for sure not info.

Most of the times, it should be a concern of user, as it's purely a property, there is no any internal state that you can expose together with the log line.

If the purpose is to find all urls being used with SimpleTargetProvider, static search would be easier.

Reply:
Honestly i don't see problem with info log as this class is singelton and will be printed only once.
You right, the main purpose  is to find all SimpleTargetProvider usages .

Reply:
You right, the main purpose is to find all urls used with SimpleTargetProvider.
I asked my self how often we are creating SimpleTargetProvider ? 
SimpleTargetProvider is created only once when we are passing TargetProvider to the client builder and will be printed only once per client.